resources/less/forum.less

```diff
@@ -19,9 +21,50 @@
 }
 
 .LogInButton--google {
```

Comment:
We might want to specify this google button to be this extensions... as technically the user could be using the extension without google enabled & another ext (or custom code) for google login, and we're imposing these styles. I don't necessarily mean change the item name, but perhaps add an extra class.